# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/svg/SVGSymbolFrame.h
# Commit: b16351df250c
# Full Hash: b16351df250c572a14efa126f7702fe104ac9fd7
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-07-22 21:26:48
# Regressor Bug: 1843945
# File Overlap Count: 1
# Description:
#   Bug 1843945 - optimise SVGObserverUtils r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183802
# ==============================================================================

diff -r 7e8063ebfd83 -r b16351df250c layout/svg/SVGSymbolFrame.h
--- a/layout/svg/SVGSymbolFrame.h	Sat Jul 22 10:03:30 2023 +0000
+++ b/layout/svg/SVGSymbolFrame.h	Sat Jul 22 14:06:38 2023 +0000
@@ -24,7 +24,9 @@
 
  protected:
   explicit SVGSymbolFrame(ComputedStyle* aStyle, nsPresContext* aPresContext)
-      : SVGViewportFrame(aStyle, aPresContext, kClassID) {}
+      : SVGViewportFrame(aStyle, aPresContext, kClassID) {
+    AddStateBits(NS_STATE_SVG_RENDERING_OBSERVER_CONTAINER);
+  }
 
  public:
   NS_DECL_QUERYFRAME
